benchmarks/measure.js

```diff
@@ -2,10 +2,7 @@
 
 const os = require('os');
 const { performance: performanceHooks } = require('perf_hooks');
-const times = require('ramda/src/times');
-const median = require('ramda/src/median');
-const map = require('ramda/src/map');
-const prop = require('ramda/src/prop');
+const { times, median, map, prop } = require('ramda');
```

Comment:
We tried to avoid loading the ramda index file because it caused a significant slowdown of ESLint startup time, see also #213.
I think the root-cause, that this plugin doesn’t work with node 17 and 18 is in ramda itself, see also #322. Unfortunately downgrading ramda seems also like a bad idea because of #319.
The fix in ramda has already been merged, but it hasn’t been released yet, see also https://github.com/ramda/ramda/issues/3278.

Reply:
Hey, so another route to take instead of waiting for `ramda`, is to switch to [`rambda`](https://github.com/selfrefactor/rambda)
The only limitation in switching to `rambda` is that there is no `memoizeWith` function, so you'll need to implement your own, which isn't difficult
I've already done the work to migrate over, which is based off this PR
Let me know if i should add it to this PR or open a separate PR

Here's a screenshot of the passing tests
<img width="938" alt="image" src="https://user-images.githubusercontent.com/3140325/167280475-5c76655d-f878-47bd-aaf4-87a5010446bd.png">

Reply:
@lo1tuma Could we push this PR? We are right now blocked on this PR for our support. The "significant slowdown" is significantly less important than supporting Node 18 at this moment.

Reply:
@jrandolf changes vs my refactor to rambda
<img width="992" alt="image" src="https://user-images.githubusercontent.com/3140325/168028151-c1463127-cd9e-47ea-9a6d-be060a6ed4ac.png">

Reply:
I'm opening a separate PR @jrandolf

Reply:
> The "significant slowdown" is significantly less important than supporting Node 18 at this moment.

@jrandolf this might be the case for you, but probably not for everyone.
Anyway, thanks for taking the effort and trying to help to resolve the issue. I hope to find some time this week to take a closer look at all options and make decision.